rtloader/common/builtins/aggregator.c

```diff
@@ -192,7 +192,7 @@ static PyObject *submit_metric(PyObject *self, PyObject *args)
     bool flush_first_value = false;
 
     // Python call: aggregator.submit_metric(self, check_id, aggregator.metric_type.GAUGE, name, value, tags, hostname, flush_first_value)
-    if (!PyArg_ParseTuple(args, "OsisdOs|i", &check, &check_id, &mt, &name, &value, &py_tags, &hostname, &flush_first_value)) {
+    if (!PyArg_ParseTuple(args, "OsisdOs|b", &check, &check_id, &mt, &name, &value, &py_tags, &hostname, &flush_first_value)) {
```

Comment:
Yes. `b` forces the parsing to be done on one byte, which happens to work for our purpose here, but it would definitely be cleaner to use standard types. We can do a follow-up PR to test if defining `flush_first_value` as `int` and parsing it as `int` would work as well.